### DISCUSSION TITLE
How to put my files to Currency converter branch

### DISCUSSION THREAD

**Discussion by JeongseopYun** [Satish-Mutyala/T24S2T1EXTG12#3] [Q&A] [answered]
Hello team, I'm sorry but I suddenly cannot remember. Could anyone be able to tell me how to merge my local file to the currency converter branch? If you can guide me on which lecture should I look at again or share some files would be much appreciated. Thank you. 

![IMG_1531](https://github.com/user-attachments/assets/8b5e2a86-e831-4068-83d9-1b6fe58dcaca)

**Comment by Satish-Mutyala** [comment, marked as answer]
@JeongseopYun  Please refer Session 04 class recording. If you can refer the Session 04 Class Exercises , Activity 03, it has examples of how to commit the code to local branch and sync and push to remote repositoty.

**Comment by WeiZhou-dev** [comment]
Hi @JeongseopYun, lecture 4 and week 4 activities will help you with this by using TortoiseGit git syn and push.
